configure.sh

```diff
@@ -59,53 +47,25 @@ while [[ "$PIP_MANYLINUX2010" == "" ]]; do
   esac
 done
 
-# CPU
-if [[ "$TF_NEED_CUDA" == "0" ]]; then
-
-  # Check if it's installed
-  if [[ $(pip show tensorflow) == *tensorflow* ]] || [[ $(pip show tf-nightly) == *tf-nightly* ]] ; then
-    echo 'Using installed tensorflow'
-  else
-    # Uninstall GPU version if it is installed.
-    if [[ $(pip show tensorflow-gpu) == *tensorflow-gpu* ]]; then
-      echo 'Already have gpu version of tensorflow installed. Uninstalling......\n'
-      pip uninstall tensorflow-gpu
-    elif [[ $(pip show tf-nightly-gpu) == *tf-nightly-gpu* ]]; then
-      echo 'Already have gpu version of tensorflow installed. Uninstalling......\n'
-      pip uninstall tf-nightly-gpu
-    fi
-    # Install CPU version
-    if [[ "$PIP_MANYLINUX2010" == "0" ]]; then
-        echo 'Installing tensorflow 1.14......\n'
-        pip install tensorflow==1.14
-    else
-        echo 'Installing tensorflow 2......\n'
-        pip install tensorflow
-    fi
-  fi
-
+# Check if it's installed
+if [[ $(pip show tensorflow) == *tensorflow* ]] || [[ $(pip show tf-nightly) == *tf-nightly* ]] || [[ $(pip show tensorflow-cpu) == *tensorflow-cpu* ]]; then
```

Comment:
this should not build against all tf-nightly.

Reply:
Note that this tf-nightly part was already in the original configure script. By default it will install normal tensorflow, but if tf-nightly was *already* installed by the user, then it will assume that the user wants to use that instead.

Do you still think we should remove it? I'm neutral on this.

Reply:
the build might fail if you build against unstable TF so I am in favour of removing it.